dotcom-rendering/src/components/EditorialButton/EditorialButton.stories.tsx

```diff
@@ -1,65 +1,18 @@
 import { css } from '@emotion/react';
```

Comment:
When updating stories, we tend to update to CSF3: https://storybook.js.org/blog/storybook-csf3-is-here/

Some example PRs:
- https://github.com/guardian/dotcom-rendering/pull/12541
- https://github.com/guardian/dotcom-rendering/pull/12540
- https://github.com/guardian/dotcom-rendering/pull/12538

Not a blocker for this PR

Reply:
Good shout! Done